subsys/nrf_security/src/drivers/cracen/cracenpsa/src/lib_kmu.c

```diff
@@ -71,8 +71,10 @@ int lib_kmu_provision_slot(int slot_id, struct kmu_src_t *kmu_src)
 		return -LIB_KMU_NULL_PNT;
 	}
 
-	/* DEST must be on a 64-bit boundary */
-	__ASSERT(IS_PTR_ALIGNED(kmu_src->dest, uint64_t), "unaligned kmu_src->dest");
+	/* DEST must be 64/128-bit aligned */
```

Comment:
Can you clarify why we have both 64/128 alignment option here? For which platform we have the second value?

Reply:
done